packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -29,6 +31,34 @@ New usage:
   }
 ```
 
+### Applications using a child design system
```

Comment:
Could we mention the old name too? Some thing like this:
`### Child design system packages (formerly site packages)`